modules/031-linstor/docs/CONFIGURATION.md

```diff
@@ -9,10 +9,24 @@ data:
   linstorEnabled: "true"
 ```
 
-The module requires no configuration and has no parameters.
-
 After enabling the module, the cluster is automatically configured to use LINSTOR, and all that remains is to configure the storage.
 
+The module requires no configuration and has no parameters. However, some functions may require a master passphrase.  
+To set a master passphrase, create a Secret in the `d8-system` namespace:
+
+```yaml
+apiVersion: v1
+kind: Secret
+metadata:
+  name: linstor-passphrase
+  namespace: d8-system
+immutable: true
+stringData:
+  MASTER_PASSPHRASE: *!passphrase* # Master passphrase for LINSTOR
+```
+
+> **Warning**: Be responsible to choose the master passphrase for LINSTOR. If it get lost, the encrypted data will be inaccessible.
```

Comment:
```suggestion
> **Warning**: Choose strong passphrase and store it securely. If it get lost, the encrypted data will be inaccessible.
```

Reply:
BTW `password` seem to be a little more common term that `passphrase`. I'd probably stick to `password`.

Reply:
This is [official term of linstor](https://linbit.com/drbd-user-guide/linstor-guide-1_0-en/#s-linstor-encrypted-volumes), I would not change it.